map.html

```diff
@@ -198,22 +200,25 @@ <h1 class="header-dark">Find food support services in Maine</h1>
 
       <div class="row">
         <div class="col-md-5 col-sm-12 col-12">
-          <label class="top-label" for="category-select">Category</label>
-          <select class="custom-select select-default my-1 mr-sm-2" 
-                  id="category-select"               
-                  name="category-select[]"
-                  multiple="multiple">
-          </select>
+          <label class="top-label" for="category-select">Category
+            <select class="custom-select select-default my-1 mr-sm-2" 
+                    id="category-select"               
+                    name="category-select[]"
+                    multiple="multiple">
+            </select>
+          </label>
         </div>
         <div class="col-md-3 col-6">
-          <label class="top-label" for="county-select">County</label>
-          <select class="custom-select select-default my-1 mr-sm-2" id="county-select">
-          </select>
+          <label class="top-label" for="county-select">County
+            <select class="custom-select select-default my-1 mr-sm-2" id="county-select">
+            </select>
+          </label>
         </div>
         <div class="col-md-4 col-6">
-          <label class="top-label" for="town-select">Town</label>
-          <select class="custom-select select-default my-1 mr-sm-2" id="town-select">
-          </select>
+          <label class="top-label" for="town-select">Town
```

Comment:
I like the wrapping format as it groups associated elements nicely, although it is semantically equivalent to a separated label in these situations. Were there reasons beyond cleaning up groupings that you prefer this format?